src/matlab2tikz.m

```diff
@@ -154,32 +154,21 @@ function matlab2tikz(varargin)
                             'Octave', struct('name','3.8', 'num',[3 8]));
     checkDeprecatedEnvironment(minimalVersion);
 
-    m2t.cmdOpts = [];
-    m2t.currentHandles = [];
-
-    m2t.transform = []; % For hgtransform groups
+    m2t.args            = []; % For command line arguments
+    m2t.current         = []; % For currently active objects
+    m2t.transform       = []; % For hgtransform groups
     m2t.pgfplotsVersion = [1,3];
-    m2t.about.name = 'matlab2tikz';
-    m2t.about.version = '1.0.0';
-    m2t.about.author = 'Nico Schlömer';
-    m2t.about.authorEmail = 'nico.schloemer@gmail.com';
-    m2t.about.years = '2008--2015';
-    m2t.about.website = 'http://www.mathworks.com/matlabcentral/fileexchange/22022-matlab2tikz-matlab2tikz';
-    m2t.about.github = 'https://github.com/matlab2tikz/matlab2tikz';
-    m2t.about.wiki = [m2t.about.github '/wiki'];
-    m2t.about.issues = [m2t.about.github '/issues'];
+    m2t.about.name      = 'matlab2tikz';
+    m2t.about.version   = '1.0.0';
+    m2t.about.years     = '2008--2016';
+    m2t.about.website   = 'http://www.mathworks.com/matlabcentral/fileexchange/22022-matlab2tikz-matlab2tikz';
```

Comment:
Hm i think we should refer to github too, as matlabcentral is not really up to date all the time or at least needs some effort to keep it updated.